SwiftUIDZ/ContentView.swift

```diff
@@ -1,21 +1,209 @@
+        static let wifiTitleText = "Wi-Fi"
+        static let bluetoothTitleText = "Bluetooth"
+        static let mobileTitleText = "Сотовая связь"
+        static let modemTitleText = "Режим модема"
+        static let vpnTitleText = "VPN"
+        static let recomendationNotificationCount = "3"
+        static let updateNotificationCount = "1"
+        static let commonCornerRadius: CGFloat = 10
+        static let commonSizeNumber: CGFloat = 30
+        static let profileSizeNumber: CGFloat = 60
+        static let userFontSizeNumber: CGFloat = 25
+        static let userDescriptionFontSizeNumber: CGFloat = 15
+        static let zeroNumber = 0
+    }
+    
+    @State private var isOnAvia = false
+    @State private var isOnVPN = false
+    @State private var wifiSelection = Constants.zeroNumber
+    @State private var bluetoothSelection = Constants.zeroNumber
+    @State private var mobileSelection = Constants.zeroNumber
+    @State private var modemSelection = Constants.zeroNumber
+    
+    private var wifiStatuses = Constants.wifiStatuses
+    private var setUpStatuses = Constants.setUpStatuses
+    private var mobileStatuses = Constants.mobileStatuses
+
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+        NavigationView {
+            Form {
+                Section {
+                    setupProfileView()
+                    setupRecomendarionView()
+                }
+                Section {
+                    setupUpdateView()
+                }
+                Section {
+                    setupAviaView()
+                    setupWiFiView()
+                    setupBluetoothView()
+                    setupMobileView()
+                    setupModemView()
+                    setupVpnView()
+                }
+            }
+            .navigationTitle(Constants.navigationTitleText)
+        }
+    }
+    
+    private func setupProfileView() -> some View {
+        HStack {
+            ZStack {
+                Image(Constants.yinYangImageName)
+                    .resizable()
+                    .frame(width: Constants.profileSizeNumber, height: Constants.profileSizeNumber)
+            }
+            VStack {
+                HStack {
+                    Text(Constants.userName)
+                        .font(.system(size: Constants.userFontSizeNumber))
+                    Spacer()
+                }
+                HStack {
+                    Text(Constants.profileTitleText)
+                        .font(.system(size: Constants.userDescriptionFontSizeNumber))
+                    Spacer()
+                }
+            }
+            Spacer()
+            Image(systemName: Constants.chevronForwardImageName)
+                .foregroundColor(.gray)
+        }
+    }
+    
+    private func setupRecomendarionView() -> some View {
```

Comment:
```suggestion
    private func setupRecomendarionView() -> some View {
```
Так же поправь везде